src/Main.java

```diff
@@ -0,0 +1,39 @@
+import animals.Animal;
+import animals.Fish;
+import animals.Pinguin;
+import animals.Shark;
+import animals.Sparrow;
+import animals.Tiger;
+import aviaries.Aquarium;
+import aviaries.AviaryForAll;
+import aviaries.CageForBirds;
+
+public class Main {
+    public static void main(String[] args) {
+        Animal pinguin = new Pinguin();
+        Pinguin pinguin2 = new Pinguin();
+        Fish pinguin3 = new Pinguin();
+        Animal tiger = new Tiger();
+        Fish shark = new Shark();
+        Sparrow sparrow = new Sparrow();
+
+        AviaryForAll aviaryForAll = new AviaryForAll();
+        Aquarium aquarium = new Aquarium();
+        CageForBirds cageForBirds = new CageForBirds();
```

Comment:
@Orbit419 тут можна використати дженерики
```suggestion
Aviary<some generic staff> aviaryForAll = new ...
Aviary<some generic staff> aquarium = new ...
Aviary<some generic staff> cageForBirds = new ...
```